### PR TITLE
Fix 1

### PR DESCRIPTION
**Pull request by Gautime**
Fixed issues - 
### 4) Add step by step guide on how to get Google credentials.

Also removed every trace of GCResult.ts after already replacing it with export async function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Integrates google calendar with your Rocket.Chat server.
 
+* **This project is for [Google Summer of Code 2019](https://summerofcode.withgoogle.com/)**.
```
Comment (d-gubert) [inline README.md]:
```suggestion
* **This project has been developed as part of the [Google Summer of Code 2019](https://summerofcode.withgoogle.com/) program**.
```

File: README.md [hunk 2]
```diff
@@ -36,6 +59,28 @@ Integrates google calendar with your Rocket.Chat server.
...
+#### Handling Errors
+
+Anytime the app is not behaving as expected or commands not working, try these 2 things - 
+
+* First, check the format of the command and make sure the format is correct (for eg. create command).
+* Try logging in again, by first logging out using `/calendar logout` command and again authenticating.
+
+**If after trying these 2 methods, commands still dont work, raise an issue in the repository, explaining the current behaviour, things you have tried.**
+
+### Wanna improve the Calendar Integration?
```
Comment (d-gubert) [inline README.md]:
```suggestion
### Want to improve the Google Calendar app?
```

File: helpers/GSGetter.ts [hunk 3]
```diff
@@ -70,34 +72,35 @@ export class GCGetter {
...
-                    await modify.getCreator().finish(message);
-                } catch (e) {
-                    this.app.getLogger().error('Failed sending logout url', e);
-                    message.setText('An error occurred when trying to send the logout url:disappointed_relieved:');
-
-                    modify.getNotifier().notifyUser(context.getSender(), message.getMessage());
+                const logout_url = 'https://accounts.google.com/o/oauth2/revoke';
+                const revoke_token = await persistence.get_access_token(context.getSender());
+                const logout_response = await http.get(`${logout_url}?token=${revoke_token}`);
+                console.log('This is revoke token response:', logout_response);
```
Comment (d-gubert) [inline helpers/GSGetter.ts]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (Gautime) [reply README.md]: Changed as suggested.

File: README.md (hunk 2 above)
Comment (Gautime) [reply README.md]: Changed as suggested.

File: helpers/GSGetter.ts (hunk 3 above)
Comment (Gautime) [reply helpers/GSGetter.ts]: Removed as suggested.

**Pull request merged**